### PR TITLE
fix raf bug

### PR DESCRIPTION
**Pull request by martingronlund**
closes https://github.com/weizhenye/ASS/issues/14

### CODE REVIEW

File: dist/ass.js [hunk 1]
```diff
@@ -1782,6 +1782,9 @@
       framing.call(this$1);
       this$1._.requestId = raf(frame);
     };
+    if (this._.requestId) {
```
Comment (weizhenye) [inline dist/ass.js]:
```suggestion
```

File: dist/ass.js [hunk 2]
```diff
@@ -1782,6 +1782,9 @@
...
+      caf(this._.requestId);
```
Comment (weizhenye) [inline dist/ass.js]:
```suggestion
    caf(this._.requestId);
```

File: dist/ass.js [hunk 3]
```diff
@@ -1782,6 +1782,9 @@
...
+    }
```
Comment (weizhenye) [inline dist/ass.js]:
```suggestion
```

File: dist/ass.js (hunk 2 above)
Comment (weizhenye) [reply dist/ass.js]: `caf(0)` will not affect anything so I think it's fine to remove `if`

**Comment by weizhenye** [comment]
I can't get the relevance of this error and your solution, do you have any way to reproduce it?

From your solution, I guess `video.play()` is called multi times and triggered multi `play` events, then... I don't know.

Anyway, it should solve some edge cases and I'm OK to merge it.

**Comment by martingronlund** [comment]
@weizhenye thanks; fixed

It's probably a bit complex to reproduce.
We have reactive code which triggers rerenders.
It happens rarely when the user is scrubbing the video around timestamp 0.

**Pull request merged**